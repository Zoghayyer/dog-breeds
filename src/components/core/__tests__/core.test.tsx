import React from 'react';
import { shallow } from 'enzyme';
import Core from '../core';

describe('(Component) Core', () => {
  let component;

  const defaultProps = {};

  const shallowCore = (extendedProps = {}) => {
    component = shallow(<Core {...defaultProps} {...extendedProps} />);
  };

  beforeEach(() => shallowCore());

  it('renders without crashing', () => {
    shallow(<Core {...defaultProps} />);
    expect(component).toMatchSnapshot();
  });
});
