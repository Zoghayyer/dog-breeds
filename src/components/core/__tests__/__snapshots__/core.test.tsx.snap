// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Core renders without crashing 1`] = `
<div
  className="container-fluid"
  id="app-content"
>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(DogsContainer)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</div>
`;
